backend/models/mitigation.js

```diff
@@ -135,6 +132,11 @@ class Mitigation {
 			mitigations[i] = newMitigation;
 		}
 
+		if (purgeOld) {
+			const clearMitigations = this.clearAll(account);
+			await clearMitigations;
```

Comment:
No need to break this into 2 lines
```suggestion
			await this.clearAll(account);
```